src/periphery/MgvCleaner.sol

```diff
@@ -3,17 +3,15 @@ pragma solidity ^0.8.10;
 
 import {MgvLib, MgvStructs} from "mgv_src/MgvLib.sol";
 import {IMangrove} from "mgv_src/IMangrove.sol";
-import "mgv_src/MgvHelpers.sol";
 
 /* The purpose of the Cleaner contract is to execute failing offers and collect
- * their associated bounty. It takes an array of offers with same definition as
- * `Mangrove.snipes` and expects them all to fail or not execute. */
+ * their associated bounty. It takes an array of offers and will attempt to clean them individually. */
 
 /* How to use:
    1) Ensure *your* address approved Mangrove for the token you will provide to the offer (`inbound_tkn`).
    2) Run `collect` on the offers that you detected were failing.
 
-   You can adjust takerWants/takerGives and gasreq as needed.
+   Instead of using your own address, you can use `collectByImpersonation` to impersonate another taker (`takerToImpersonate`) who has approved Mangrove for `inbound_tkn`. This allows borrowing that taker's `inbound_tkn` funds for cleaning instead of using `msg.sender`'s funds (who need not have any).
```

Comment:
takerToImpersonate is now just `taker`, right?

Reply:
Indeed, I'll fix.